library/src/normal/AndroidManifest.xml

```diff
@@ -26,11 +26,11 @@
                 <action android:name="android.hardware.usb.action.USB_DEVICE_ATTACHED" />
             </intent-filter>
         </activity>
-        <activity
-            android:name=".AboutActivity"
-            android:exported="false"
-            android:parentActivityName=".MainActivity"
-            android:theme="@style/Theme.AppCompat.DayNight" />
+<!--        <activity-->
+<!--            android:name=".AboutActivity"-->
+<!--            android:exported="false"-->
+<!--            android:parentActivityName=".MainActivity"-->
+<!--            android:theme="@style/Theme.AppCompat.DayNight" />-->
```

Comment:
Unless there's comment noting about commenting this activity line, might as well just remove it.
```suggestion
```